src/lib/dialog/dialog-container.html

```diff
@@ -1 +1,3 @@
-<ng-template cdkPortalOutlet></ng-template>
+<div class="mat-dialog-component-host">
```

Comment:
Is this class meant to be both in the template and added manually?

Reply:
Yeah, we need to add it through both to cover both template and dialog portals. The class is used to propagate the `max-height` down correctly since we've got a lot of nested divs.